deps/GraphBLAS/Source/GB_control.h

```diff
@@ -354,62 +354,62 @@
 // using them will be slower.
 
 // also needed by GrB_reduce to vector
-// #define GxB_NO_FIRST_INT8    1
-// #define GxB_NO_FIRST_INT16   1
-// #define GxB_NO_FIRST_INT32   1
-// #define GxB_NO_FIRST_INT64   1
-// #define GxB_NO_FIRST_UINT8   1
-// #define GxB_NO_FIRST_UINT16  1
-// #define GxB_NO_FIRST_UINT32  1
+#define GxB_NO_FIRST_INT8    1
+#define GxB_NO_FIRST_INT16   1
+#define GxB_NO_FIRST_INT32   1
+#define GxB_NO_FIRST_INT64   1
+#define GxB_NO_FIRST_UINT8   1
+#define GxB_NO_FIRST_UINT16  1
+#define GxB_NO_FIRST_UINT32  1
```

Comment:
if a type been disabled e.g. `#define GxB_NO_FP32 1`
all of the associated semirings, bin/unary operations using that type are disabled as well.

Reply:
👍